src/pheval_exomiser/runner.py

```diff
@@ -22,6 +29,67 @@ class ExomiserPhEvalRunner(PhEvalRunner):
     config_file: Path
     version: str
 
+    def install(self, overwrite_existing: bool = False):
+        # Parse configuration options
```

Comment:
Could you please remove the inline comments for cleaner and more maintainable code?